app/services/doc_auth/lexis_nexis/error_generator.rb

```diff
@@ -0,0 +1,105 @@
+module DocAuth
+  module LexisNexis
+    class UnknownTrueIDError < StandardError; end
+
+    class ErrorGenerator
+      # These constants are the key names for the TrueID errors hash that is returned
+      ID = 'id'.to_sym
+      FRONT = 'front'.to_sym
+      BACK = 'back'.to_sym
```

Comment:
Why are these `'id'.to_sym` and not `:id`?

```suggestion
      ID = :id
      FRONT = :front
      BACK = :back
```